client/spa/js/instructor/spec/instructor.model.spec.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+/*
+global jasmine, describe, it, expect, beforeEach, afterEach, xdescribe, xit,
+spyOn
+*/
+// Get the code you want to test
+var Model = require('../instructor.model');
+
+// Test suite
+console.log('test instructor.model');
+describe('Instructor model ', function(){
+  var model;
+
+  describe('when creating a new model ', function(){
+    beforeEach(function(){
+      model = new Model();
+    });
+
+    xit('initializes with custom logic', function(){
+    });
+  });
+
+  describe('when updating the model for instructor ', function(){
+    var errorSpy;
+
+    beforeEach(function(){
```

Comment:
This is even better:

  describe('when updating the model for instructor ', function(){


    beforeEach(function(){
    
      model = new Model({
        id: 1,
        firstName: 'foo',
        lastName: 'foo',
        skills: 'foo'
      });
  
    });

    it('does not save when firstName is empty ', function(){
      model.set('firstName', null);
      model.save();
      expect(model.validationError).toEqual(['firstName cannot be empty']);
    });